{"status": "broken", "attachments": [{"source": "54d838c7-6adf-4edf-b7f3-c5cb361f88c3-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "4154f0ff-6af5-41e5-8c77-74b149060dd4-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "ca710844-bb3a-4298-a545-1101c989371d-attachment.txt", "type": "text/plain", "name": "stderr"}], "labels": [{"name": "suite", "value": "test_case_api"}, {"name": "host", "value": "DESKTOP-CLD22EL"}, {"name": "thread", "value": "18228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "test_case_api"}], "stop": 1560331397903, "uuid": "90727d9e-28af-4665-9563-b80d500a67b1", "historyId": "f093559aa765615b6cc5b6a2b1520812", "start": 1560331397798, "fullName": "test_case_api#test_kyc_open_accounts", "statusDetails": {"message": "UnicodeEncodeError: 'ascii' codec can't encode characters in position 428-433: ordinal not in range(128)", "trace": "def test_kyc_open_accounts():\n    \n        print(\"------------------------------------开户---------------------------\")\n        paths = \"/merchant/kyc.htm\"\n        nums = random.choice(\"python\")\n        pam = {\"partnerId\": global_parameter.PARTNERID,\n               \"accountType\": 1,\n               \"area\": \"中国\",\n               \"country\": \"中国\",\n               \"province\": \"上海\",\n               \"address\": \"上海市上海\",\n               \"city\": \"上海\",\n               \"name\": \"企业法人\",\n               \"idCard\": 588596854788889652,\n               \"idCardImg1\": 241354094869118976,\n               \"idCardImg2\": 241354094869118976,\n               \"merName\": \"api开户\"+nums,\n               \"businessNumber\": 45821,\n               \"business\": 241354094869118976}\n    \n        r_obj = Public_Request()\n        r_resu = r_obj.public_request(pas=paths,parameters=pam)\n>       print(json.dumps(r_resu.json(), indent=2, sort_keys=False, ensure_ascii=False))\nE       UnicodeEncodeError: 'ascii' codec can't encode characters in position 428-433: ordinal not in range(128)\n\ntest_case_api.py:199: UnicodeEncodeError"}, "name": "test_kyc_open_accounts"}